ui/src/home-module/home-page/home-page.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Banners } from 'src/Shared/models/Banners';
+import { Categories } from 'src/Shared/models/Categories';
+import { HomeServiceService } from 'src/Shared/services/home-service.service';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.sass']
+})
+export class HomePageComponent implements OnInit {
+
+  constructor(private homeService: HomeServiceService, private router: Router) { }
+  bannerData: Banners[] = [];
+  categoriesData: Categories[] = [];
+  ngOnInit(): void {
+    if (sessionStorage.getItem('isLoggedIn') && sessionStorage.getItem('isLoggedIn') === 'true') {
```

Comment:
Shouldn't this be part of some `auth` or `login` service?

Reply:
Yeah. Solved this issue now.